docs/gpu-feature-discovery/README.md

```diff
@@ -209,6 +209,7 @@ their meaning:
 | nvidia.com/gpu.machine         | String     | Machine type                                 | DGX-1          |
 | nvidia.com/gpu.memory          | Integer    | Memory of the GPU in Mb                      | 2048           |
 | nvidia.com/gpu.product         | String     | Model of the GPU                             | GeForce-GT-710 |
+| nvidia.com/gpu.mode            | String     | Display or Compute Mode of the GPU           | compute        |
```

Comment:
Could we provide more information on what this value means?

Reply:
We still need to add a reference as to what this means. From the documentation it is also only applicable to specific device types. Do we want to link to the relevant documentation here?

Reply:
FIxed